### PR TITLE
Add wreeto to Wikis

### PR DESCRIPTION
**Pull request by chrisvel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1120,6 +1120,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [turndown](https://domchristie.github.io/turndown/) - HTML to Markdown converter written in JavaScript. ([Source Code](https://github.com/domchristie/turndown)) `MIT` `Javascript`
 - [Trilium Notes](https://github.com/zadam/trilium) - Trilium Notes is a hierarchical note taking application with focus on building large personal knowledge bases. `AGPL-3.0` `Nodejs`
 - [Turtl](https://turtl.it/) - Totally private personal database and note taking app. ([Source Code](https://github.com/turtl)) `GPL-3.0` `CommonLisp`
+- [Wreeto](https://github.com/chrisvel/wreeto_official) - Wreeto is an open source note-taking, knowledge management and wiki system built on top of Ruby on Rails framework. `AGPL-3.0` `Ruby`
```
Comment (epsilon-0) [inline README.md:1123]: Change the link for Wreeto to point towards the main page of wreeto and add a source code field at the end.

It is nice to show users a demo of what you made.

Comment (nodiscc) [reply README.md:1123]: @chrisvel Please update the main link to https://wreeto.com/, and add a Source code link pointing to https://github.com/chrisvel/wreeto_official

Comment (chrisvel) [reply README.md]: OK! Thank you.

**Comment by chrisvel** [comment]
> I'm not a member, I just helped a tiny bit.
> 
> My approval means nothing, nor can I give it :P

I'm sorry, I saw your nick in reviewers and I thought there is a policy for repository owners.

**Comment by chrisvel** [comment]
Who can merge it then? 

> Only those with **write** access to this repository can merge pull requests.

**Pull request merged**